### DISCUSSION TITLE
stuck on [INFO] Starting go2rtc healthcheck service...

### DISCUSSION THREAD

**Discussion by muttsan** [blakeblackshear/frigate#6692] [General]
So it is stuck on [INFO] Starting go2rtc healthcheck service...   The webui will not load,  only have 1 camera in config, and nothing else.
It has not shut down the docker container still shows as running it is just sitting there.

**Comment by NickM-27** [comment]
That's the last line of the logs in a normal startup so that doesn't give any clues. More info is needed to be able to help

**Comment by muttsan** [comment]
sorry that is the last it shows, that is where it stops,  but I have been playing using the same command line but not using compose and switching to docker run it works,  docker-compose is not
